app/src/features/datasets/DatasetPage.tsx

```diff
@@ -42,6 +42,197 @@ const useStyles = makeStyles((theme: Theme) => ({
   }
 }));
 
+const rawTemplate = `
```

Comment:
super nit: Maybe name this something more descriptive like: `SIMSHandlerbarsTemplate`

I know we only plan to have this hardcoded temporarily, but just in case